sdk/eventhub/Azure.Messaging.EventHubs/tests/Snippets/Sample09_ObservableEventBatchLiveTests.cs

```diff
@@ -0,0 +1,220 @@
+            var eventHubName = scope.EventHubName;
+#endif
+
+            var producer = new EventHubProducerClient(connectionString, eventHubName);
+
+            try
+            {
+                using var eventBatch = await producer.CreateBatchAsync();
+                ObservableEventDataBatch newbatch = new ObservableEventDataBatch(eventBatch);
+
+                for (var index = 0; index < 5; ++index)
+                {
+                    var eventBody = new BinaryData($"Event #{ index }");
+                    var eventData = new EventData(eventBody);
+
+                    if (!newbatch.TryAdd(eventData))
+                    {
+                        throw new Exception($"The event at { index } could not be added.");
+                    }
+                }
+
+                foreach (var singleEvent in newbatch.Events)
+                {
+                    Debug.WriteLine($"Added event { singleEvent.EventBody } at time { singleEvent.EnqueuedTime }");
+                }
+
+                await producer.SendAsync(newbatch);
+            }
+            finally
+            {
+                await producer.CloseAsync();
+            }
+
+#endregion
+        }
+
+        /// <summary>
+        ///   Performs basic smoke test validation of the contained snippet.
+        /// </summary>
+        ///
+        [Test]
+        public async Task Sample09_CheckingBatch()
+        {
+            await using var scope = await EventHubScope.CreateAsync(1);
+
+            #region Snippet:Sample09_CheckingBatch
+
+#if SNIPPET
+            var connectionString = "<< CONNECTION STRING FOR THE EVENT HUBS NAMESPACE >>";
+            var eventHubName = "<< NAME OF THE EVENT HUB >>";
+#else
+            var connectionString = EventHubsTestEnvironment.Instance.EventHubsConnectionString;
+            var eventHubName = scope.EventHubName;
+#endif
+
+            var producer = new EventHubProducerClient(connectionString, eventHubName);
+
+            try
+            {
+                using var eventBatch = await producer.CreateBatchAsync();
+                ObservableEventDataBatch newbatch = new ObservableEventDataBatch(eventBatch);
+
+                for (var index = 0; index < 5; ++index)
+                {
+                    var eventBody = new BinaryData($"Event #{ index }");
+                    var eventData = new EventData(eventBody);
+                    eventData.Properties.Add("ApplicationId", index);
+
+                    if (!newbatch.TryAdd(eventData))
+                    {
+                        throw new Exception($"The event at { index } could not be added.");
+                    }
+                }
+
+                // Verify that the expected event is in the batch
+                var contains = newbatch.Events.Any(eventData => int
+                .TryParse(eventData.Properties["ApplicationId"].ToString(), out var id) && id == 1);
```

Comment:
```suggestion
                // Verify that the expected event is in the batch
                var contains = newbatch.Events
                    .Any(eventData => int.TryParse(eventData.Properties["ApplicationId"].ToString(), out var id) && id == 1);
```

Indentation and line breaking was a bit unusual.   We should double-check to see how this looks rendered in GitHub to ensure that we're not scrolling off the screen, but I think we're safe with this form.